website/docs/cloud-docs/api-docs/organization-tags.mdx

```diff
@@ -40,6 +40,14 @@ description: >-
 
 This API returns the list of tags used with all resources across the organization. Tags can be added to this pool directly or via those resources. Tags deleted here will be removed from all other resources. Tags can be added, applied, removed and deleted in bulk.
 
+Here are some things to note about tags:
+
+- Workspace tags or tags must be one or more characters; can include letters, numbers, colons, hyphens, and underscores; and must begin and end with a letter or number.
+- Tags have a 255 character limit.
```

Comment:
I think we can nicely combine this into my suggested sentence above since they are both about tag name  restrictions
```suggestion
```